src/components/editor/PropertyLabel.jsx

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+import shortid from 'shortid'
+import React, {Component} from 'react'
+import { OverlayTrigger, Popover } from 'react-bootstrap'
+import PropTypes from 'prop-types'
+import RequiredSuperscript from './RequiredSuperscript'
+
+export class PropertyLabel extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  render () {
+
+    let title = []
```

Comment:
I believe you should use `const` here since you're not reassigning `title` below.